addons/mail/static/src/messaging/entity/composer/composer.js

```diff
@@ -50,6 +84,27 @@ function ComposerFactory({ Entity }) {
             });
         }
 
+        /**
+         * @param mentionKeyword
+         */
+        insertSuggestedPartner(mentionKeyword) {
+            const selection = this.suggestedPartners.find(suggestedPartner => suggestedPartner == this.activePartner);
```

Comment:
cc @alexkuhn here I would consider making the method generic, so that the partner to insert is passed as param.
This would allow "on click" to directly pass the correct partner instead of having to write on "activePartner" first.

Reply:
Totally make sense for such a named method to have `partner` as its parameter instead of `mentionKeyword`. This is especially true when invoking this method comes after `get_mention_suggestions` that just populated partners in store.

Reply:
And you shouldn't need `mentionKeyword` because you can just rely on the 1st occurrence of `@` on the left starting from cursor start.

Reply:
Done